### DISCUSSION TITLE
Build Errors

### DISCUSSION THREAD

**Discussion by iwl** [procount/pinn#764] [General]
.[7m>>> pigz 2.7 Downloading.[27m                                                                                                                                                                                  http://zlib.net/pigz/pigz-2.7.tar.gz ...... Not Found.
http://sources.buildroot.net/pigz-2.7.tar.gz ...... Not Found.
make: *** [package/pkg-generic.mk:75: xyz/pinn/buildroot/output/build/pigz-2.7/.stamp_downloaded] Error 1

http://sources.buildroot.net/pigz/pigz-2.7.tar.gz  (manually downloaded, to fix somewhere)

**Comment by procount** [comment]
They seem to have upgraded pigz from 2.7 to 2.8 and removed 2.7.
Try changing the version number in buildroot/package/pigz/pigz.mk.
